README.md

```diff
@@ -347,6 +360,14 @@ Ignore breakpoints in files stored in `node_modules` folder or ending in `.test.
 ```
 $ node-debug --hidden node_modules/ --hidden \.test\.js$ app
 ```
+Only show files with the word 'test.js' in the path:
+```
+$ node-debug --hidden=^\(?\!.*test\.js.*\) app
+```
+Only show files with the word 'test.js' or 'hello.js' in the path:
+```
+$ node-debug --hidden=^\(?\!\(.*test\.js.*\)\|\(.*hello\.js.*\)\) app
+```
```

Comment:
I like the idea behind these  new examples as they shows how to implement a white-list using the black-list-based option `--hidden`.

However, instead of escaping each character, the example should enclose the regexp in single quotes, e.g.

```
--hidden='^(?!.*test\.js.*)'
'--hidden=^(?!.*test\.js.*)'
```